pages/en/docs/guides/security/index.md

```diff
@@ -410,6 +410,13 @@ The use of experimental features in production isn't recommended.
 Experimental features can suffer breaking changes if needed, and their
 functionality isn't securely stable. Although, feedback is highly appreciated.
 
+## OpenSSF Tools
+
+The [OpenSSF][] is leading several initiatives that can be very useful, especially if you plan to publish an Npm package. These initiatives include:
```

Comment:
Maybe 'Npm' to be better in 'npm' ("N" in a lower case more natural)
or just use ```npm```?

Reply:
branding-wide, it’s always npm, lowercase, and never Npm for sure.